.eslintrc.json

```diff
@@ -51,20 +51,16 @@
       "error",
       "global"
     ],
-    "@typescript-eslint/ban-types": [
-      "error",
-      {
-        "types": {
-          "Function": false
-        }
-      }
-    ],
+    "@typescript-eslint/no-unsafe-function-type": "off",
     "@typescript-eslint/no-unsafe-member-access": "off",
     "@typescript-eslint/no-unsafe-assignment": "off",
     "@typescript-eslint/no-unsafe-return": "off",
     "@typescript-eslint/no-unsafe-argument": "off",
     "@typescript-eslint/no-unsafe-call": "off",
     "@typescript-eslint/no-unsafe-enum-comparison": "off",
+    // Note: you must disable the base rule as it can report incorrect errors
+    "no-unused-expressions": "off",
+    "@typescript-eslint/no-unused-expressions": "off",
```

Comment:
Literally hundreds of failures here, many of them in tests from mocha assertions (i.e., expect(...)... returns an assertion, so it is technically an unused expression).

We could just disable this in our test suite, but for consistency I disabled it everywhere.

Reply:
If we've disabled 'no-unused-expressions,' is there a reason we need to remove the unused `error`s from the test files?

Reply:
Different lint rules 🙂 the `errors` are removed from no-unused-vars, not no-unused-expressions.

Reply:
ahh i see!